fe/fe-core/src/main/java/com/starrocks/alter/MaterializedViewHandler.java

```diff
@@ -468,7 +468,10 @@ private List<Column> checkAndPrepareMaterializedView(CreateMaterializedViewStmt
                 if (mvColumnItem.isKey()) {
                     ++numOfKeys;
                 }
-                Preconditions.checkNotNull(baseColumn, "Column[" + mvColumnName + "] does not exist");
+                Preconditions.checkNotNull(baseColumn,
+                        "The materialized view column of aggregation or unique or primary table " +
```

Comment:
nit:should here carry mvColumnName information?
The materialized view column[" + mvColumnName + "] of